### ISSUE TITLE
Some icons are partially copied from Windows

### ISSUE THREAD

**Issue by MrRar** [SerenityOS/serenity#17468]
Here is a comparison of some of the icons I noticed that look the same:

![cerenity](https://user-images.githubusercontent.com/16406771/218556251-71099d87-4af3-48d5-a5bf-a96db2822e38.png)

This not an exhaustive list there are many more.

Not sure where Serenity gets icons from but they are definitely copied.

**Comment by kleinesfilmroellchen** [comment]
> Not sure where Serenity gets icons from but they are definitely copied.
Contributors submit icons via PRs normally, and it is expected that these icons are original or license-compatible.

I'm not a lawyer, but most of these icons are transformed enough to probably not fall under copyright (you can't copyright the look of a flip notebook at 16x16 px resolution). As icons are not consistent either way this definitely needs to be fixed though.

**Comment by MrRar** [comment]
> I'm not a lawyer, but most of these icons are transformed enough to probably not fall under copyright (you can't copyright the look of a flip notebook at 16x16 px resolution).

Yeah, I don't know about the legal stuff. But morally speaking you should at least credit Microsoft for contributing to the icons.

**Comment by mulquin** [comment]
The second sentence of the about section is "It flatters with sincerity by stealing beautiful ideas from various other systems."

I don't think this is a problem until someone from Microsoft considers it a problem.

**Comment by mulquin** [comment]
I don't think the project has ever denied the obvious inspiration of Windows from that time period. The second sentence of the about section is "It flatters with sincerity by stealing beautiful ideas from various other systems."

**Comment by kleinesfilmroellchen** [comment]
Let's end this pointless discussion about legality and morality around copying icons; the truth is that there is no good unified style of icons and these are the first ones that need to be adjusted.

**Comment by kleinesfilmroellchen** [comment]
> Maybe, a custom license information could be added for the icons, where it is said that they are similiar to the Microsoft icons.

No, that is a non-goal.

> This [new icon](https://github.com/SerenityOS/serenity/pull/17475), in my opinion, looks more like Windows XP.

Luckily your opinion does not constitute rule of law. More seriously: inspiration, especially from an era-related operating system, is something that Serenity already has and will continue to have.
